data/libs/SpaceStation.lua

```diff
@@ -697,6 +697,17 @@ Event.Register("onLeaveSystem", function (ship)
 	if ship ~= Game.player then return end
 	destroySystem()
 end)
+
+Event.Register("onShipDestroyed", function (ship, _)
+	for _,local_police in pairs(police) do
+		for k,police_ship in pairs(local_police) do
+			if (v == police_ship) then
```

Comment:
Maybe I'm not thinking straight, but `v` isn't defined, right? It should be `ship` instead?